store/store.go

```diff
@@ -70,6 +74,20 @@ func (s *storeImpl) Get(key string) (interface{}, bool) {
 	return object.contents, true
 }
 
+// GetByRegex returns objects from the store based on a regexp search.
+func (s *storeImpl) GetByRegex(expr string) (m map[string]interface{}) {
+	s.mutex.RLock()
+	defer s.mutex.RUnlock()
+
+	m = make(map[string]interface{})
```

Comment:
then the comment is misleading, it says `GetByRegex returns objects from the store based on a regexp search.` i had impression you need to return objects